pinot-core/src/test/java/org/apache/pinot/queries/ForwardIndexDisabledMultiValueQueriesTestWithReload.java

```diff
@@ -0,0 +1,525 @@
+/**
```

Comment:
Any reason we can't add these tests to the existing ForwardIndexDisabledMultiValueQueriesTest.java file? Just like you did for SingleValueQueriesTest?

Reply:
+1

Reply:
Yes, there are only 2 MV columns in the avro file used in these tests. Today I disable them both (2 segments for 1 column and 1 segment for the other). It was very tricky to rewrite those tests to both keep my existing tests and add these new reload tests as I would need the `BeforeMethod` to essentially setup the segment differently for reload vs. non-reload tests. Or I would have to remove the forward index disabled flag for one of the columns for the non-reload tests and rewrite those tests.

This gave me more flexibility to also disable dictionary for the second column etc.

Let me know if you prefer that I consolidate the two and rewrite some of the test cases in the file to only assess forward index disabled for 1 column instead of two.